### PR TITLE
Reworked load options

### PR DESCRIPTION
**Pull request by halvardssm**
Hi @crowlKats ! I needed some extra options for my own application, and thought you might want to use them as well. Let me know if you want something changed or discuss the option changes!

### CODE REVIEW

File: mod.ts [hunk 1]
```diff
@@ -16,13 +25,28 @@ export function parse(string: string) {
...
+  path = ".env",
+  priorityEnv = false,
+  ignoreMissingFile = false,
+}: LoadOptions) {
+  let file;
+
+  try {
+    file = await Deno.readFile(path);
+  } catch (e) {
+    if (ignoreMissingFile) return;
```
Comment (crowlKats) [inline mod.ts]:
```suggestion
    if (ignoreMissingFile && e instanceof Deno.errors.NotFound) return;
```

**Review by crowlKats — APPROVED** [review]
LGTM, thanks for this!
I nearly forgot I even have this module

**Comment by halvardssm** [comment]
I did the changes and ran the formatter 💪🏻

**Comment by crowlKats** [comment]
Thanks @halvardssm!

**Pull request merged**